ocp_resources/datavolume.py

```diff
@@ -229,3 +240,52 @@ def _check_none_pending_status(self, failure_timeout=120):
         except TimeoutExpiredError:
             self.logger.error(f"{self.name} status is {sample}")
             raise
+
+    def is_garbage_collector_enabled_dv(self):
+        if self.exists:
+            dv_annotations = self.instance.metadata.get("annotations")
+            if dv_annotations:
+                return (
+                    dv_annotations.get("cdi.kubevirt.io/storage.deleteAfterCompletion")
+                    is not False
+                )
+        else:
+            if self.pvc.exists and self.pvc.instance.status.phase == "Bound":
```

Comment:
`return self.pvc.exists and self.pvc.instance.status.phase == "Bound"`

Reply:
ack